diff --git a/datefudge.c b/datefudge.c
index fe93ef8..5c2db2e 100644
--- a/datefudge.c
+++ b/datefudge.c
@@ -66,8 +66,8 @@ time_t time(time_t *x) {

 #endif

-int __gettimeofday(struct timeval *x, struct timezone *y) {
-    static int (*libc_gettimeofday)(struct timeval *, struct timezone *) = NULL;
+int __gettimeofday(struct timeval *x, void *y) {
+    static int (*libc_gettimeofday)(struct timeval *, void *) = NULL;
     int res;

     if(!libc_gettimeofday)
@@ -78,7 +78,7 @@ int __gettimeofday(struct timeval *x, struct timezone *y) {
     return 0;
 }

-int gettimeofday(struct timeval *x, struct timezone *y) {
+int gettimeofday(struct timeval *x, void *y) {
     return __gettimeofday(x,y);
 }
